tests/test_micropub.php

```diff
@@ -490,17 +491,46 @@ function test_create_location_h_adr() {
 		$this->assertEquals( '', get_post_meta( $post->ID, 'geo_address', true ) );
 	}
 
-	function test_create_location_plain_text() {
```

Comment:
did you mean to delete the `_plain_text` test? we still want it, right?

Reply:
Accident. Restoring it.